content/en/docs/concepts/traffic-management/index.md

```diff
@@ -13,237 +13,156 @@ aliases:
+[network resilience and testing features](#network-resilience-and-testing) that
+are built in to the API’s resources.
+
+## Virtual services {#virtual-services}
+
+[Virtual services](/docs/reference/config/networking/v1alpha3/virtual-service/),
+along with destination rules, are the key building blocks of Istio’s traffic
+routing functionality. A virtual service lets you configure how requests are
+routed to a service within an Istio service mesh, building on the basic
+connectivity and discovery provided by Istio and your platform. Each virtual
+service consists of a set of routing rules that are evaluated in order, letting
+Istio match each given request to the virtual service to a specific real
+destination within the mesh. Your mesh can require multiple virtual services or
+none depending on your use case.
+
+### Why use virtual services? {#why-use-virtual-services}
+
+Virtual services play a key role in making Istio’s traffic management flexible
+and powerful. They do this by strongly decoupling where the client sends their
+requests from the destination workloads that actually implement them. Virtual
+services also provide a rich way of specifying different traffic routing rules
+for sending traffic to those workloads.
+
+So why is this so useful? Without virtual services, Envoy just distributes
+traffic using round-robin load balancing between all service instances, as
+described in the introduction. You can improve this behavior with what you know
+about the workloads: for example, some might represent a different version. This
+can be useful in A/B testing where you might want to configure traffic routes
+based on percentages across different service versions, or you might want to use
+a different load balancing policy for a particular set of instances.
+
+With virtual services, you create a new hostname which directs traffic to
+services of your choice, or overrides traffic behavior for an existing hostname.
+You can then use routing rules in the virtual service that tell Envoy how to
+send the virtual host’s traffic to the appropriate "real" workloads.
+
+A typical use case is to send traffic to different versions of a service,
+specified as service subsets. Clients send requests to the virtual service as if
+it was a single entity, and Envoy then routes the traffic to the different
+versions depending on the virtual service rules (for example "20% of calls go to
+the new version" or "calls from this URI go to version 2"). This allows you to,
+for instance, create a canary rollout where you gradually increase the
+percentage of traffic that’s sent to a new service version. The traffic routing
+is completely separate from the instance deployment, meaning that the number of
+instances implementing the new service version can scale up and down based on
+traffic load without referring to traffic routing at all. By contrast, container
+orchestration platforms like Docker or Kubernetes support canary rollouts, but
```

Comment:
suggest:

 By contrast, container orchestration platforms like Kubernetes only support traffic distribution based on instance scaling, which quickly becomes complex.

Docker is not really an orchestration platform, and I wouldn't consider K8s supporting canary rollouts -- and saying they do takes away from the Istio value add

Reply:
Oh I like that. Will change.

Reply:
The current doc has a link to [my old blog post](https://archive.istio.io/v1.1/blog/2017/0.1-canary/) that better explains the difference with orchestration platform support. We added the link from the concepts because it was the only doc we had on the subject and because it's an old blog post users were having trouble finding it. I think we should still have a link unless we move the material from the blog into the docs somewhere that is easier to find.

Reply:
I'll put the link back in, that content is very useful. I'll have a think about where we might put it in the main docset.